docs/source/functional.rst

```diff
@@ -8,145 +8,176 @@ torchaudio.functional
 
 Functions to perform common audio operations.
 
-:hidden:`spectrogram`
-~~~~~~~~~~~~~~~~~~~~~
+:hidden:`Filtering`
```

Comment:
the functionals aren't moved in 0.7 but this subsection isn't visible, right?

Reply:
It is visible. `hidden` here is a command to tell doxygen which class it should use to render this section.
Filtering subsection is for the improved user experience and has nothing to do with where the location these functions are defined.

Reply:
Right, this seems reasonable